src/main/asyncapi/network_survey_messaging.yaml

```diff
@@ -1665,6 +1825,45 @@ components:
       description: The estimated horizontal accuracy of the provided location, radial, in meters. We define horizontal accuracy as the radius of 68% confidence. In other words, if you draw a circle centered at this location's latitude and longitude, and with a radius equal to the accuracy, then there is a 68% probability that the true location is inside the circle. A value of 0 indicates there was no available horizontal accuracy.
       example: 40
 
+    rotationX:
+      type: number
+      format: float
+      minimum: -90.0
+      maximum: 90.0
+      description: The rotation reading contains 3 angles, measured in degrees that define the orientation of the device in three-dimensional space. These angles are similar to yaw, pitch and roll but are defined using only right hand rotation with axes as defined by the right hand cartesian coordinate system. The three angles are applied to the device in the following order. Right-handed rotation z (-180, 180]. Starting from the y-axis and incrementing in the counter-clockwise direction. Right-handed rotation x [-90, 90]. Starting from the new (once-rotated) y-axis and incrementing towards the z-axis. Right-handed rotation y (-180, 180]. Starting from the new (twice-rotated) z-axis and incrementing towards the x-axis.
+      example: 24
+
+    rotationY:
+      type: number
+      format: float
+      minimum: -180.0
+      maximum: 180.0
+      description: The rotation reading contains 3 angles, measured in degrees that define the orientation of the device in three-dimensional space. These angles are similar to yaw, pitch and roll but are defined using only right hand rotation with axes as defined by the right hand cartesian coordinate system. The three angles are applied to the device in the following order. Right-handed rotation z (-180, 180]. Starting from the y-axis and incrementing in the counter-clockwise direction. Right-handed rotation x [-90, 90]. Starting from the new (once-rotated) y-axis and incrementing towards the z-axis. Right-handed rotation y (-180, 180]. Starting from the new (twice-rotated) z-axis and incrementing towards the x-axis.
+      example: 24
+
+    rotationZ:
+      type: number
+      format: float
+      minimum: -180.0
+      maximum: 180.0
+      description: The rotation reading contains 3 angles, measured in degrees that define the orientation of the device in three-dimensional space. These angles are similar to yaw, pitch and roll but are defined using only right hand rotation with axes as defined by the right hand cartesian coordinate system. The three angles are applied to the device in the following order. Right-handed rotation z (-180, 180]. Starting from the y-axis and incrementing in the counter-clockwise direction. Right-handed rotation x [-90, 90]. Starting from the new (once-rotated) y-axis and incrementing towards the z-axis. Right-handed rotation y (-180, 180]. Starting from the new (twice-rotated) z-axis and incrementing towards the x-axis.
```

Comment:
For all of these rotation fields, it might be best to include that this is for the antenna. The description is great and is very detailed on the specifics of this value, but it would be helpful to include in the first sentence or two a high level view of "Why would I want to use this value".